src/services/users.services.ts

```diff
@@ -0,0 +1,14 @@
+
+class UsersService {
+  public getUser(username: string) {
+    return new Promise((resolve, reject) => {
```

Comment:
Please use a return type. What's the method return?
Should be something like: `public async getUser(username: string): Promise<User> {...}`